pkg/csi_driver/controller_test.go

```diff
@@ -52,6 +52,7 @@ func initTestController(t *testing.T) csi.ControllerServer {
 		driver:      initTestDriver(t),
 		fileService: fileService,
 		cloud:       cloudProvider,
+		volumeLocks: util.NewVolumeLocks(),
```

Comment:
We should be able to add some simple tests to run through the volume locking code paths, no?

Reply:
Ack